boards/nrf51dk/src/main.rs

```diff
@@ -269,7 +284,7 @@ pub unsafe fn reset_handler() {
     chip.systick().reset();
     chip.systick().enable(true);
 
-    debug!("Initialization complete. Entering main loop");
+    debug!("Initialization complete. Entering main loop\r\n");
```

Comment:
Looks like `debug!` does `\n`, nor `\r\n`: https://github.com/helena-project/tock/blob/master/kernel/src/debug.rs#L263

@alevy I can add the `\r` if needed? (separate issue from this PR)